app/src/main/java/co/touchlab/kampkit/android/ui/Composables.kt

```diff
@@ -0,0 +1,155 @@
+package co.touchlab.kampkit.android.ui
+
+import androidx.compose.animation.Crossfade
+import androidx.compose.animation.core.FastOutSlowInEasing
+import androidx.compose.animation.core.TweenSpec
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import co.touchlab.kampkit.android.R
+import co.touchlab.kampkit.db.Breed
+import co.touchlab.kampkit.models.DataState
+import co.touchlab.kampkit.models.ItemDataSummary
+
+@Composable
+fun DogList(breeds: List<Breed>, onItemClick: (Breed) -> Unit) {
+    LazyColumn {
+        items(breeds) { breed ->
+            DogRow(breed) {
+                onItemClick(it)
+            }
+            Divider()
+        }
+    }
+}
+
+@Composable
+fun DogRow(breed: Breed, onClick: (Breed) -> Unit) {
+    Row(
+        Modifier
+            .clickable { onClick(breed) }
+            .padding(10.dp)
+    ) {
+        Text(breed.name, Modifier.weight(1F))
+        FavoriteIcon(breed)
+    }
+}
+
+@Composable
+fun FavoriteIcon(breed: Breed) {
+    Crossfade(
+        targetState = breed.favorite == 0L,
+        animationSpec = TweenSpec(
```

Comment:
Haha thanks! It's kinda wild how easy animation is in Compose. Was thinking it could be fun to make custom Swipe Refresh indicator like here: https://github.com/google/accompanist/blob/main/sample/src/main/java/com/google/accompanist/sample/swiperefresh/SwipeRefreshCustomIndicatorSample.kt#L98-L101, maybe like a spinning dog head something dog themed. Could do an Instagram-style expanding heart animation on likes, too. 😄